docs/content/users/install/docker-installation.md

```diff
@@ -22,10 +22,10 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     colima start --cpu 4 --memory 6 --disk 100 --vm-type=qemu --mount-type=sshfs --dns=1.1.1.1
     ```
     (On macOS versions before Ventura, omit the `--vm-type=qemu` flag as it doesn't work on older OS versions.)
-    
+
     After the initial run above, you can use `colima start` or use `colima start -e` to edit the configuration file. Run `colima status` at any time to check Colima’s status.
 
-    When your computer restarts, you’ll need to `colima start` again. This will eventually be automated in later versions of Colima.
+    When your computer restarts, you’ll need to `colima start` again. Starting with Colima 0.6.0 you are able to automate the startup process. `brew services start colima` adds Colima to the list of login items for your macOS user account. With `brew services list` you are able to verify if the service's property list file was correctly added to `~/Library/LaunchAgents`.
```

Comment:
```suggestion
    When your computer restarts, you’ll need to `colima start` again. If you prefer Colima to start automatically on reboot, in Colima `v0.6+` use `brew services start colima` to configure auto-start.
```

Reply:
You would strike the part about `brew services list` again? But i am ok with removing it again. The only i am unsure about is using Colima twice. would something like that work? 

```
If you prefer Colima to start automatically on reboot, use `brew services start colima` in version 0.6+ to configure auto-start.
```

Reply:
People don't really need to know anything except the `brew services start` part. It gets pretty wordy otherwise. Nobody knows what a plist is or cares, for example

Reply:
i agree (i think the only person who was interested in that `brew services list` detail was me probably) . and would my rephrased suggestion work?

Reply:
Yes, minor changes:

> If you prefer to start Colima automatically on reboot, use `brew services start colima` in Colima version 0.6+ to configure auto-start. 

should work.

Reply:
I've pushed the changes with the version you've quoted. i'll set the conversation to resolved.